posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,600 @@
+       <feature>jakartaeeClient-10.0</feature>
+ </featureManager>
+----
+For more information reference:
+
+* https://jakarta.ee/specifications/platform/10/[Jakarta EE Platform 10], https://jakarta.ee/specifications/webprofile/10/[Jakarta EE Web Profile 10], and link:https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE Core Profile 10] specifications. 
+
+
+* link:{url-prefix}/docs/latest/reference/javadoc/liberty-jakartaee10-javadoc.html[Jakarta EE 10 Javadoc]
+
+
+
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/24582
+// Contact/Reviewer: ReeceNana,Emily-Jiang
+// // // // // // // // 
+[#mp6]
+== MicroProfile 6.0   
+
+MicroProfile continues to innovate how the industry optimizes Java microservices.  The MicroProfile 6.0 release enables applications to use MicroProfile APIs together with <<core, Jakarta EE Core Profile 10>> along with various other new functionality and improvements. The complete list of specifications that make up MicroProfile 6.0 is as follows:
+
+
+[cols="4,2,3"]
+|===
+|Specification |Updates |Liberty Feature Documentation
+
+[[telemetry]]
+|https://github.com/eclipse/microprofile-telemetry/releases/tag/1.0[MicroProfile Telemetry 1.0]
+|*New*
+|link:{url-prefix}/docs/latest/reference/feature/mpTelemetry-1.0.html[mpTelemetry-1.0]
+
+[[metrics]]
+|https://github.com/eclipse/microprofile-metrics/releases/tag/5.0[MicroProfile Metrics 5.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/mpMetrics-5.0.html[mpMetrics-5.0]
+
+[[openapi]]
+|https://github.com/eclipse/microprofile-open-api/releases/tag/3.1[MicroProfile OpenAPI 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/mpOpenAPI-3.1.html[mpOpenAPI-3.1]
+
+[[jwt]]
+|https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.1[MicroProfile JWT Authentication 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/mpJwt-2.1.html[mpJwt-2.1]
+
+|https://github.com/eclipse/microprofile-config/releases/tag/3.0.2[MicroProfile Config 3.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpConfig-3.0.html[mpConfig-3.0]
+
+|https://github.com/eclipse/microprofile-health/releases/tag/4.0.1[MicroProfile Health 4.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpHealth-4.0.html[mpHealth-4.0]
+
+|https://github.com/eclipse/microprofile-rest-client/releases/tag/3.0.1[MicroProfile Rest Client 3.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpRestClient-3.0.html[mpRestClient-3.0]
+
+|https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/4.0.2[MicroProfile Fault Tolerance 4.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpFaultTolerance-4.0.html[mpFaultTolerance-4.0]
+
+|https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE Core Profile 10]
+
+|New
+|<<coreprofile, See Core Profile table>>

```

Comment:
Suggest making Core Profile first in the list as that is what everything else is built on top of

Reply:
I struggled with the order.  I decided to lead with MP Telemetry since it's an MP spec rather then leading with a Jakarta spec in the MP section.  
I'll delay this change for now.